massa-consensus/src/settings.rs

```diff
@@ -271,6 +275,7 @@ pub struct ConsensusConfig {
 }
 
 lazy_static::lazy_static! {
+    /// Compact representation of key values of consensus algorithm used in API
     static ref CONFIG: Config = Config {
```

Comment:
just "config" may be a confusing name

Reply:
this is a STATIC_CONFIG :1st_place_medal:

Reply:
Compact config ?